### PR TITLE
200 and 404 tests passing

### PR DESCRIPTION
**Pull request by SamR232**

### CODE REVIEW

File: models/articles.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ exports.updateVotesByArticleId = (articleId, inc_votes) => {
     )
     .then((results) => {
       if (results.rows.length === 0) {
-        return Promise.reject({ status: 404, msg: "article not found" });
+        return Promise.reject({ status: 404, msg: "article not found" });git
```
Comment (HeatherEver) [inline models/articles.js]: Think this 'git' at the end of the line might have been added my mistake?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -115,22 +115,31 @@ describe("PATCH /api/articles/:article_id", () => {
...
+    expect(body.msg).toEqual("article not found");
+  });
+});
+
+describe("GET api/users", () => {
+  test("Status: 200 - responds with an array of objects with a username property", async () => {
+    const { body } = await request(app).get("/api/users");
+    expect(200);
+    expect(body).toBeInstanceOf(Array);
+    expect(typeof body[0].username).toBe("string");
```
Comment (HeatherEver) [inline __tests__/app.test.js:139]: Could be worth adding an expect to check all the keys in the object are correct and another to check the length of the array!

**Review by HeatherEver — CHANGES_REQUESTED** [review]
Good work! Just a couple of extra expect statements to pop in!

File: models/articles.js (hunk 1 above)
Comment (SamR232) [reply models/articles.js]: removed!

File: __tests__/app.test.js (hunk 2 above)
Comment (SamR232) [reply __tests__/app.test.js:139]: I think the only expected key is usernames?

**Pull request closed without merging**